broadcaster/lib/broadcaster_web/controllers/media_controller.ex

```diff
@@ -72,15 +82,15 @@ defmodule BroadcasterWeb.MediaController do
   end
 
   def event_stream(conn, %{"resource_id" => resource_id}) do
-    case PeerSupervisor.fetch_pid(resource_id) do
-      {:ok, _pid} ->
-        conn
-        |> put_resp_header("content-type", "text/event-stream")
-        |> put_resp_header("connection", "keep-alive")
-        |> put_resp_header("cache-control", "no-cache")
-        |> send_chunked(200)
-        |> update_layers()
-
+    with {:ok, _pid} <- PeerSupervisor.fetch_pid(resource_id),
+         [stream_id, _pc_id] <- String.split(resource_id, "-") do
```

Comment:
To be fully correct, `resource_id` is in fact `pc_id` so after `String.splti` we get `stream_id` and some postifx. or at least that's how I understand it looking at line 43 where as `resource_id`, we return `pc_id`, which in turn has embeded `stream_id`. That might sound like a nitpick but for someone who was not writing the code, it is a bit missleading :stuck_out_tongue: